src/format.iced

```diff
@@ -0,0 +1,122 @@
+stream = require('keybase-chunk-stream')
+util = require('./util')
+
+space = Buffer.from(' ')
+newline = Buffer.from('\n')
+punctuation = Buffer.from('.')
+
+words_per_line = 200
+chars_per_word = 15
+
+noop = () ->
+
+exports.FormatStream = class FormatStream extends stream.ChunkStream
+
+  __format : (chunk) ->
+    return Buffer.concat([space, chunk])
+
+  _format : (chunk) ->
+    if chunk.length < chars_per_word then return chunk
```

Comment:
It won't, because this function will only ever be passed a chunk that satisfies `chunk.length < chars_per_word` precisely once, per the contract of the `ChunkStream` class (in the `node-chunk-stream` repo)